### DISCUSSION TITLE
Time tracking only 24h

### DISCUSSION THREAD

**Discussion by mediakanal** [YetiForceCompany/YetiForceCRM#14250] [General]
Hello, I would like to be able to set the time recording module so that you can record a data record for longer than 24 hours.

**Comment by mariuszkrzaczkowski** [comment]
https://gitdeveloper.yetiforce.com/index.php?parent=Settings&module=EventHandler&view=Index
![image](https://user-images.githubusercontent.com/10198654/108188201-9862ec00-710f-11eb-8235-a53a3f436dd9.png)

**Reply by mediakanal** [reply]
thanks
